lib/explorer/backend/lazy_series.ex

```diff
@@ -105,6 +105,8 @@ defmodule Explorer.Backend.LazySeries do
     count: 1,
     nil_count: 1,
     skew: 2,
+    corr: 3,
+    cov: 2,
```

Comment:
I would call it `correlation` and `covariance`, since they are short names and we usually prefer to avoid abbreviations. But let's hear what José thinks :)

Reply:
Agreed!